db/schema.rb

```diff
@@ -465,8 +465,8 @@
     t.string "city_code"
     t.string "post_code"
     t.string "city_name"
-    t.enum "invited_through", default: "unknown", enum_name: "user_invited_through"
-    t.enum "created_through", default: "unknown", enum_name: "user_created_through"
+    t.enum "invited_through", default: "unknown", enum_type: "user_invited_through"
+    t.enum "created_through", default: "unknown", enum_type: "user_created_through"
```

Comment:
Oh, en fait ce changement c’est [en prévision](https://github.com/bibendi/activerecord-postgres_enum/commit/9410fe9526b5a06529fe3c16a15ac99c7e9cf671) de Rails 7 [qui gèrera les enums PG nativement](https://blog.saeloun.com/2022/01/12/rails-7-adds-custom-enum-support-in-postgresql.html).